### ISSUE TITLE
Pressing F5 on a breakpoint also pushes the key into the emulator

### ISSUE THREAD

**Issue by Yazwh0** [indigodarkwolf/box16#80]
Looks like pressing F5 (to run) on a breakpoint also pushes the keystroke into the emulator. (Keyup maybe?)

![breakissue](https://user-images.githubusercontent.com/90315107/214963627-d6f149c3-5198-4756-8002-1a8abc281c37.gif)

**Comment by mooinglemur** [comment]
I don't think this bug has existed since the emulator display was placed in a dockable ImGui window.

**Comment by indigodarkwolf** [comment]
Closed for now, then.
